File: z_cs_py_yapilanlar/CFlags.py
class CFlags:
    def __init__(self):
        self.reset()

    def __del__(self):
        pass

    def initialize(self, Sistem):
        return self

    def reset(self):
        self.BakiyeGuncelle = False
        self.KomisyonGuncelle = False
        self.DonguSonuIstatistikGuncelle = False
        self.KomisyonuDahilEt = True
        self.KaymayiDahilEt = False
        self.AnlikKarZararHesaplaEnabled = False
        self.KarAlYuzdeHesaplaEnabled = False
        self.IzleyenStopYuzdeHesaplaEnabled = False
        self.ZararKesYuzdeHesaplaEnabled = False
        self.KarAlSeviyeHesaplaEnabled = False
        self.ZararKesSeviyeHesaplaEnabled = False
        self.AGerceklesti = False
        self.SGerceklesti = False
        self.FGerceklesti = False
        self.PGerceklesti = False
        self.IdealGetiriHesapla = True
        self.IdealGetiriHesaplandi = False
        return self

File: z_cs_py_yapilanlar/CFlags.cs
public class CFlags
{
    public bool BakiyeGuncelle = false;
    public bool KomisyonGuncelle = false;
    public bool DonguSonuIstatistikGuncelle = false;
    public bool KomisyonuDahilEt = true;
    public bool KaymayiDahilEt = false;

    public bool AnlikKarZararHesaplaEnabled = false;
    public bool KarAlYuzdeHesaplaEnabled = false;
    public bool IzleyenStopYuzdeHesaplaEnabled = false;
    public bool ZararKesYuzdeHesaplaEnabled = false;
    public bool KarAlSeviyeHesaplaEnabled = false;
    public bool ZararKesSeviyeHesaplaEnabled = false;

    public bool AGerceklesti = false;
    public bool SGerceklesti = false;
    public bool FGerceklesti = false;
    public bool PGerceklesti = false;

    public bool IdealGetiriHesapla = false;
    public bool IdealGetiriHesaplandi = false;

    ~CFlags()
    {

    }

    public CFlags()
    {

    }

    public CFlags Initialize(dynamic Sistem)
    {
        return this;
    }

    public CFlags Reset(dynamic Sistem)
    {
        BakiyeGuncelle = false;
        KomisyonGuncelle = false;
        DonguSonuIstatistikGuncelle = false;
        KomisyonuDahilEt = true;
        KaymayiDahilEt = false;

        AnlikKarZararHesaplaEnabled = false;
        KarAlYuzdeHesaplaEnabled = false;
        IzleyenStopYuzdeHesaplaEnabled = false;
        ZararKesYuzdeHesaplaEnabled = false;
        KarAlSeviyeHesaplaEnabled = false;
        ZararKesSeviyeHesaplaEnabled = false;

        AGerceklesti = false;
        SGerceklesti = false;
        FGerceklesti = false;
        PGerceklesti = false;

        IdealGetiriHesapla = true;
        IdealGetiriHesaplandi = false;

        return this;
    }
}